procedures/unit-testing-tracing.ipf

```diff
@@ -280,26 +284,33 @@ static Function WriteProcList(string procWinList)
 
 	numProcs = ItemsInList(procWinList)
 
-	newCode = "Make/FREE/T wt = {\\\r"
-	for(i = 0; i < numProcs - 1; i += 1)
-		newCode += "\"" + StringFromList(i, procWinList) + "\",\\\r"
-	endfor
-	newCode += "\"" + StringFromList(numProcs - 1, procWinList) + "\"\\\r"
-	newCode += "}\r"
-	newCode += "return wt\r"
-	WAVE/T wNewCode = ListToTextWave(newCode, "\r")
+	if(numProcs)
```

Comment:
If you fix bugs in old code put that in a separate commit.

Reply:
After reordering of commits this is outdated.